### PR TITLE
[PSDK][KERNEL32] Export functions needed for Syzkaller

### PR DESCRIPTION
**Pull request by Freakston**
This Pull Request has the required modifications to run Syzkaller.

### CODE REVIEW

**Comment by SergeGautherie** [comment]
Wow! NT6 functions in NT5.2 `kernel32`?
Should use `kernel32_vista` instead?

**Comment by Freakston** [comment]
> Wow! NT6 functions in NT5.2 `kernel32`?
> Should use `kernel32_vista` instead?

That isn't working as expected and this particular build will only be used for fuzzing purposes. @SergeGautherie

File: sdk/include/psdk/winbase.h [hunk 1]
```diff
@@ -1362,11 +1362,11 @@ typedef enum _COMPUTER_NAME_FORMAT {
 	ComputerNamePhysicalDnsFullyQualified,
 	ComputerNameMax
 } COMPUTER_NAME_FORMAT;
-#endif /* (_WIN32_WINNT >= 0x0500) */
-
-#if (_WIN32_WINNT >= 0x0600)
+ /* (_WIN32_WINNT >= 0x0500) */
+//Modified by Freakston for Syzkaller
```
Comment (ThFabba) [inline sdk/include/psdk/winbase.h:1366]: If you're building it within our build system... could you just define _WIN32_WINNT to be a higher version and link to kernel32_vista?

**Comment by SergeGautherie** [comment]
> this particular build

Ah, this PR is not meant to be committed?
Then, you should be more explicit about its actual purpose and mark it as draft?

**Comment by binarymaster** [comment]
> Ah, this PR is not meant to be committed?

You probably didn't noticed that this PR is special GSoC PR:
image

**Comment by SergeGautherie** [comment]
> special GSoC PR

What difference does that make?

**Pull request closed without merging**